access/pom.xml

```diff
@@ -70,6 +70,20 @@
 					</systemProperties>
 				</configuration>
 			</plugin>
+			
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-war-plugin</artifactId>
+				<version>3.1.0</version>
+				<configuration>
+					<webResources>
+						<resource>
+							<filtering>true</filtering>
+							<directory>../etc</directory>
+						</resource>
+					</webResources>
+				</configuration>
+			</plugin>
```

Comment:
One other thing to add, if you look in admin/pom.xml in the jetty plugin, you'll see some webapp specific config.  Some of the settings significantly improve startup time my making it smarter about where to get jsp and other files from:
```
<webApp>
    <webInfIncludeJarPattern>.*access-common-.*</webInfIncludeJarPattern><allowDuplicateFragmentNames>true</allowDuplicateFragmentNames>
    <extraClasspath>${basedir}/target/test-classes/</extraClasspath>
</webApp>
```